### PR TITLE
Change Date Created to Published Date, ref #645

### PR DESCRIPTION
**Pull request by awead**

### CODE REVIEW

File: spec/factories/works.rb [hunk 1]
```diff
@@ -41,18 +41,6 @@
       end
     end
 
-    trait :full_meta_data do
```
Comment (carolyncole) [inline spec/factories/works.rb:44]: @awead not sure why you are removing this trait.  That seems a bit weird.

**Review by carolyncole — CHANGES_REQUESTED** [review]
@awead looks good with the exception of removing the trait.

File: spec/factories/works.rb (hunk 1 above)
Comment (awead) [reply spec/factories/works.rb:44]: @cam156 it's not used.

Comment (carolyncole) [reply spec/factories/works.rb:44]: @awead it is not really related to this commit, but it does look like it should have been removed in this commit or an earlier one( https://github.com/psu-stewardship/scholarsphere/commit/ca52ec1f5a74441168ef17ac61bd6a61da3381e5#diff-8011362fff9e5db5c9997382f4418f28  ) Just making sure we are not loosing test coverage.

**Review by carolyncole — APPROVED** [review]

**Pull request merged**